gallery/gallery/lib/pages/demo.dart

```diff
@@ -190,6 +191,7 @@ class _DemoPageState extends State<DemoPage> with TickerProviderStateMixin {
         },
       ),
       actions: [
+        SizedBox(width: appBarPadding),
```

Comment:
I don't think this SizedBox is necessary.

Reply:
It's just to keep things balanced/centered